dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -1032,6 +1034,14 @@
 		io-channels = <&adc3 18>;
 		status = "disabled";
 	};
+
+	vref: vref {
+		compatible = "st,stm32-vref";
+		vrefint-cal-addr = <0x1FF1E860>;
+		vrefint-cal-mv = <3300>;
+		io-channels = <&adc3 19>;
```

Comment:
To continue on my previous comment, this specific property should be redefined in stm32h723.dtsi and stm32h7a3.dtsi, to select the correct ADC and channel.

Reply:
Please see my previous comment.